wicket-auth-roles/src/test/java/org/apache/wicket/authroles/authorization/strategies/role/annotations/AnnotationsRoleAuthorizationStrategyTest.java

```diff
@@ -16,52 +16,209 @@
  */
 package org.apache.wicket.authroles.authorization.strategies.role.annotations;
 
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
 import org.apache.wicket.Component;
 import org.apache.wicket.authroles.authorization.strategies.role.IRoleCheckingStrategy;
 import org.apache.wicket.authroles.authorization.strategies.role.Roles;
 import org.apache.wicket.markup.html.WebComponent;
-import org.junit.Assert;
+import org.apache.wicket.request.resource.IResource;
 import org.junit.Test;
 import org.mockito.Mockito;
 
 /**
  * Tests for {@link AnnotationsRoleAuthorizationStrategy}
  */
-public class AnnotationsRoleAuthorizationStrategyTest extends Assert
+public class AnnotationsRoleAuthorizationStrategyTest
 {
-	/**
-	 * https://issues.apache.org/jira/browse/WICKET-3974
-	 */
```

Comment:
I like these references to the ticket introduced the test.
Sometimes `git blame` doesn't help so fast.